.github/workflows/ci.yml

```diff
@@ -313,13 +313,17 @@ jobs:
     timeout-minutes: 120
     env:
       FILE_ENV: "./ci/test/00_setup_env_native_asan.sh"
+      DANGER_CI_ON_HOST_CACHE_FOLDERS: 1
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Set Ccache directory
         run: echo "CCACHE_DIR=${RUNNER_TEMP}/ccache_dir" >> "$GITHUB_ENV"
 
+      - name: Set base root directory
+        run: echo "BASE_ROOT_DIR=${RUNNER_TEMP}" >> "$GITHUB_ENV"
```

Comment:
This was made as a step and not in the `env:` section of the job as ${{ runner.temp }} is only available in the run steps.

Reply:
`RUNNER_TEMP` was used over `${{ github.workspace }}` (as is used in the MacOS job) because the CI container mounts that as readonly which means it can't be used to write cache items to.